eng/common/scripts/GeneralReleaseNotesLogic.ps1

```diff
@@ -0,0 +1,128 @@
+. (Join-Path $PSScriptRoot common.ps1)
+
+$PATTERN_REGEX = "^\[\w+(\.(?<SectionName>\w+))?\]:\s#\s\((?<Pattern>.*)\)"
+$CsvMetaData = Get-CSVMetadata
+
+function Get-PackagesInfoFromFile ($releaseNotesLocation) 
+{
+    $releaseNotesContent = Get-Content -Path $releaseNotesLocation
+    $checkLine = $False
+    $presentPkgInfo = @()
+
+    foreach ($line in $releaseNotesContent)
+    {
+        if ($line -eq "<!--")
+        {
+            $checkLine = $True
+            continue
+        }
+        if ($line -eq "-->")
+        {
+            break
+        }
+        if ($checkLine)
+        {
+            $pkgInfo = ($line.Trim()).Split(":")
+            $packageName = $pkgInfo[0]
+            $packageMetaData = $CsvMetaData | Where-Object { $_.Package -eq $packageName }
+            if ($packageMetaData.Count -gt 0)
+            {
+                $presentPkgInfo += $line.Trim()
+            }
+        }
+    }
+    return $presentPkgInfo
+}
+
+function Filter-ReleaseHighlights ($releaseHighlights)
```

Comment:
Is this still the release highlights? Shouldn't it be the package list or something along those lines?

Reply:
This is coming from the collect changelog. This is the new information that needs to be added.